src/WebApiToTypeScript/Interfaces/InterfaceService.cs

```diff
@@ -334,7 +334,7 @@ private List<MemberWithCSharpType> GetMembers(TypeDefinition typeDefinition)
                 });
 
             var properties = typeDefinition.Properties
-                .Where(p => p.GetMethod != null && p.GetMethod.IsPublic && !p.IsSpecialName)
+                .Where(p => p.GetMethod != null && p.GetMethod.IsPublic && !p.IsSpecialName && !Helpers.HasCustomAttribute(p, "JsonIgnoreAttribute"))
```

Comment:
I like the idea, but I am fine with this being here for now, since it is the most occuring case, and we can figure out a scheme for enhanced attribute support/handling as a separate issue?